data/fieldlist_NCAR.jsonc

```diff
@@ -57,6 +57,12 @@
       "scalar_coord_templates": {"plev": "Z{value}"},
       "ndim": 4
     },
+    "Z500": {
```

Comment:
Jess is correct -- see EOF_500hPa, MJO_teleconnection or temp_extremes_distshape for examples of level extraction for geopotential_height. When using the NCAR naming convention, this will look for a variable named "Z500" (where "500" is the requested pressure level in mb), and if that's not available, will try to extract the level from a 3D variable named "Z3".

Reply:
Thanks, this is the kind of feedback I was looking for. I need to look at the code tomorrow when I'm at work to see if I understand exactly the method you are requesting.